src/sentry/ingest/consumer_v2/ingest.py

```diff
@@ -0,0 +1,64 @@
+from typing import Any, Mapping
+
+import msgpack
+from arroyo.backends.kafka.consumer import KafkaPayload
+from arroyo.types import Message
+
+from sentry.ingest.ingest_consumer import (
+    _load_event,
+    _store_event,
+    process_attachment_chunk,
+    process_individual_attachment,
+    process_userreport,
+)
+from sentry.models import Project
+from sentry.utils import metrics
+
+DecodedMessage = Mapping[str, Any]
+
+
+def process_ingest_message(message: Message[KafkaPayload]) -> None:
+    """
+    Processes a single Kafka Message containing an Event payload.
+
+    This does:
+    - Decode the Kafka payload which is in msgpack format and has a bit of
+      metadata like `type` and `project_id`.
+    - Fetch the corresponding Project from cache.
+    - Further processing by `type`:
+    ...
+    - Decode the actual event payload which is in JSON format and perform some
+      initial loadshedding on it.
+    - Store the JSON payload in the event processing store, and pass it on to
+      `preprocess_event`, which will schedule a followup task such as
+      `symbolicate_event` or `process_event`.
+    """
+
+    raw_payload = message.payload.value
+    message: DecodedMessage = msgpack.unpackb(raw_payload, use_list=False)
+
+    message_type = message["type"]
+    project_id = message["project_id"]
+
+    # TODO: we are only fetching a single project, so should we use a different metric?
+    with metrics.timer("ingest_consumer.fetch_projects"):
+        project = Project.objects.get_from_cache(id=project_id)
+    projects = {project.id: project}
+
+    if message_type == "event":
+        result = _load_event(message, projects)
+        if result is not None:
+            data, callback = result
+            callback(_store_event(data))
+
+    elif message_type == "attachment_chunk":
+        process_attachment_chunk(message, projects)
+    elif message_type == "attachment":
+        process_individual_attachment(message, projects)
+    elif message_type == "user_report":
+        process_userreport(message, projects)
```

Comment:
I'd disable this code and raise an exception instead. Let's get this consumer working for just events and transactions first. 

Streaming team also suggested that we move away from multiplexing consumers. In this case we can have a simplified version for profiles, events, and transactions (without attachments) that just deals with event payloads. That's the one I'd test here.